fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMaterializedViewRefreshProcessor.java

```diff
@@ -520,7 +520,7 @@ private boolean needToRefreshNonPartitionTable(Table partitionTable) {
     }
 
     private boolean supportRefreshByPartition(Table table) {
-        if (table.isLocalTable() || table.isHiveTable() || table.isLakeTable()) {
+        if (table.isLocalTable() || table.isHiveTable() || table.isCloudNativeTable()) {
```

Comment:
```suggestion
        if (table.isLocalTable() || table.isHiveTable() || table. isCloudNativeTableOrMaterializedView()) {
```

Reply:
maybe a bug

Reply:
or `if (table.isNativeTableOrMaterializedView() || table.isHiveTable())`

Reply:
IMO we should fix this in a separate patch.